src/main/java/org/zone/utils/Messages.java

```diff
@@ -844,4 +844,12 @@ public static Component getLeavingDisplaySuccessfullyChangedToBossBar() {
                         "boss bar").color(NamedTextColor.AQUA);
     }
 
+    public static Component getNoGroupWithTheGroupKey(Identifiable groupKey) {
+        return Component.text("No Group found with the GroupKey " + groupKey.getName()).color(NamedTextColor.RED);
+    }
+
+    public static Component getNoInviteFoundByUUID() {
+        return Component.text("No invite found by your UUID");
```

Comment:
As mentioned before, players don't want to know what a UUID. This message should probably be 

``You have not been invited`` 

or something like that

Reply:
oh whoops